lib/health_monitor/providers/delayed_job.rb

```diff
@@ -70,7 +68,7 @@ def check_failures!
       end
 
       def check_failed_latency!
-        oldest = job_class.order(:run_at).where('last_error is not null').first
+        oldest = job_class.where('last_error is not null').order(:run_at).first
```

Comment:
I switched these because it is easier to stub and also more efficient SQL to get a subset then sort it.

Reply:
👍